src/shopDetail/ShopDetail.tsx

```diff
@@ -54,6 +54,30 @@ export const ShopDetail = ({ route }: TProps) => {
     );
   }, [redeemStatus?.message, redeemError?.endUserMessage, refetchRedeemStatus, t]);
 
+  const renderRedeemStatus = useCallback(() => {
+    /* 
+      When it's an online reward show either a redeembutton or the errormessage on why it's not redeemable
+      Also show a loading indicator when it's an online reward
+      When it's a physical reward, only show the errormessage on why it's not redeemable, no loader or no redeem button
+    */
+    return reward?.online ?
```

Comment:
ik ben in een render functie dan ook wel fan van if functies, voor de duidelijkheid, maar dat is misschien wel PP